pynvim/plugin/host.py

```diff
@@ -161,8 +181,10 @@ def _load(self, plugins):
                     has_script = True
                 else:
                     directory, name = os.path.split(os.path.splitext(path)[0])
-                    file, pathname, descr = find_module(name, [directory])
-                    module = imp.load_module(name, file, pathname, descr)
+                    try:
+                        module = handle_import(directory, name)
+                    except ImportError:
+                        return
```

Comment:
pynvim remote plugin stopped working with handler/autocmd not found errors, but it was being silently ignored here

Reply:
Thanks for noticing that. Good thing we didn't release yet :D Any chance you can send a fix?

Reply:
I managed to fix this. I'm sending a PR very soon.